CONTRIBUTING.md

```diff
@@ -1,16 +1,21 @@
 # CONTRIBUTION GUIDELINES
+======
 
 ## Before contributing
+------
```

Comment:
Not needed - GitHub automatically adds that in h1, and h2 headings.

Reply:
thank you for pointing it out. will remove.

Reply:
adjusted